apps/test/unit/templates/sectionsRefresh/AdvancedSettingTogglesTest.jsx

```diff
@@ -1,101 +1,73 @@
-import {shallow} from 'enzyme'; // eslint-disable-line no-restricted-imports
+import {render, screen} from '@testing-library/react';
```

Comment:
@lfryemason I updated this test after Kaitie reviewed. Anything you think should be done differently?